packages/tasks/src/model-libraries.ts

```diff
@@ -508,6 +508,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"magi-1": {
+		prettyLabel: "MAGI-1",
+		repoName: "MAGI-1",
+		repoUrl: "https://github.com/SandAI-org/MAGI-1",
+		countDownloads: `path_extension:"json"`,
```

Comment:
Doing this will likely double-count many downloads since there are several .json files needed to instantiate a model. What we usually do is to track downloads on a single file that is essential for the model to be loaded correctly. Could https://huggingface.co/sand-ai/MAGI-1/blob/main/ckpt/vae/config.json be used for that for instance?

Reply:
Thanks for your advice and I have changed the path to `ckpt/vae/config.json`